README.md

```diff
@@ -758,16 +758,25 @@ To enable business-time globally in Laravel, set default openning hours and holi
 
 ```php
 <?php return [
-  'opening-hours' => [
-    'monday' => ['08:00-12:00', '14:00-19:00'],
-    'wednesday' => ['09:00-19:00'],
-  ],
-  'holidaysAreClosed' => true,
-  'holidays' => [
+    'monday' => ['09:00-12:00', '13:00-18:00'],
+    'tuesday' => ['09:00-12:00', '13:00-18:00'],
+    'wednesday' => ['09:00-12:00'],
+    'thursday' => ['09:00-12:00', '13:00-18:00'],
+    'friday' => ['09:00-12:00', '13:00-20:00'],
+    'saturday' => ['09:00-12:00', '13:00-16:00'],
+    'sunday' => [],
+    'exceptions' => [
+        '2016-11-11' => ['09:00-12:00'],
+        '2016-12-25' => [],
+        '01-01' => [], // Recurring on each 1st of january
+        '12-25' => ['09:00-12:00'], // Recurring on each 25th of december
+    ],
+    'holidaysAreClosed' => true,
+    'holidays' => [
     'region' => 'us',
     'with' => [
-      'boss-birthday' => '09-26',
-      'last-monday'   => '= last Monday of October',
+        'boss-birthday' => '09-26',
+        'last-monday' => '= last Monday of October',
     ],
   ],
 ];
```

Comment:
This config is read by https://github.com/kylekatarnls/business-time/pull/29/files#diff-04c6e90faac2675aa89e2176d2eec7d8R760-R782

So everything outisde 'opening-hours' or 'holidays' won't be taken into account by BusinessTime (nor BusinessDay).
But however, `holidaysAreClosed` should be inside the `'opening-hours'` array.

Reply:
I do not think that is correct, BusinessDay seems to read the 'monday' -> 'sunday' attributes and process them into business days correctly.

Reply:
I use the `'opening-hours'` and it works fine. But it seems to be an issue with the holidays.

Reply:
I will fix it and keep both syntaxes for backward compatibility.

Either:
```php
<?php return [
    'opening-hours' => [
        'monday' => [
            '09:00-12:30',
            '14:00-18:30',
        ],
        'tuesday' => [
            '09:00-12:30',
            '14:00-18:30',
        ],
    ],
    'holidaysAreClosed' => true,
    'holidays' => [
        'region' => 'us',
        'with' => [
            'boss-birthday' => '07-07',
        ],
    ],
];
```

or:
```php
<?php return [
    'monday' => [
        '09:00-12:30',
        '14:00-18:30',
    ],
    'tuesday' => [
        '09:00-12:30',
        '14:00-18:30',
    ],
    'holidaysAreClosed' => true,
    'holidays' => [
        'region' => 'us',
        'with' => [
            'boss-birthday' => '07-07',
        ],
    ],
];
```